src/index.js

```diff
@@ -35,14 +35,13 @@ const agileSDK = (params) => {
   }
   // parse url to remove any irregularites
   const parsed = parseUrl(params.api);
-  const apiBase = `${parsed.origin}/api`;
+  const apiBase = params.api.indexOf('/') === 0 ? `${params.api}/api` : `${parsed.origin}/api`;
```

Comment:
This wouldn't work as expected if you pass the protocol right? It's better than it was but worth putting a not in the readme/docs.

Reply:
Could you please elaborate?
In case you pass `http://google.com`, then `params.api.indexOf('/') === 0` ends up being false, so it ends up with the same result as before.
The introduced change will only have an effect if the uri is `/api/...`

Do you suggest modifying anything, or can this be merged?